flopy/mf6/mfmodel.py

```diff
@@ -825,6 +825,12 @@ def check(self, f=None, verbose=True, level=1):
         """
         Check model data for common errors.
 
+        Warning
+        -------
+        The MF6 check mechanism may be removed for FloPy 3.10+. The
+        checks API will remain in place, but may temporarily cease
+        to function. Checks will be reimplemented for FloPy 4.x.
```

Comment:
Wonder if we should just say that checks will be reimplemented **in a future release** (here and elsewhere)?

Reply:
that is better, will do.

Reply:
changed to 

```
The MF6 check mechanism is deprecated pending reimplementation
in a future release. While the checks API will remain in place
through 3.x, it may be unstable, and will likely change in 4.x.
```

how does that sound?

Reply:
Sounds good.